docs/queries/overview.mdx

```diff
@@ -148,7 +148,7 @@ const getPosts = async () => {
     where: query // ensure that `qs` adds the `where` property, too!
   }, { addQueryPrefix: true });
 
-  const response = await fetch(`http://localhost:3000/api/posts${stringifiedQuery}`);
+  const response = await fetch(`http://localhost:3000/api/posts?${stringifiedQuery}`);
```

Comment:
This is incorrect. The `qs` package is taking care of adding the `?` to the URL param with the option `{ addQueryPrefix: true }`.

Reply:
My bad!